app/src/main/kotlin/cz/covid19cz/erouska/ui/sandbox/SandboxVM.kt

```diff
@@ -111,7 +134,7 @@ class SandboxVM(
                 L.d("files=${files}")
                 return@runCatching files.size
             }.onSuccess {
-                lastDownload.value = prefs.lastKeyExportFileName()
+                lastDownload.value = prefs.lastKeyExportFileName() +" "+ prefs.lastKeyExportTime()
```

Comment:
Pro testování nestačí jenom zobrazovat poslední download, ale seznam všech downloadů v čase nebo aspoň posledních 5.

Reply:
Dává mi smysl přidat seznam timestamp, ale nevím jestli má smysl vždy ke každé takové timestamp přidat i název posledního downloadu

Reply:
Jestli se to tam vejde, tak proč ne